changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,23 @@ Junipernetworks Junos Collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
```

Comment:
@rohitthakur2590 @justjais Major change section is required for major release changeling.

Reply:
updated minor section and galaxy.yaml. https://github.com/ansible-collections/junipernetworks.junos/pull/154